OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -738,11 +742,169 @@ private WorkshopDTO WithWorkshop()
         };
     }
 
-    private List<WorkshopDTO> WithWorkshops()
+    private WorkshopBaseDTO WithWorkshopCreate()
     {
-        return new List<WorkshopDTO>()
+        return new WorkshopBaseDTO()
         {
-            new WorkshopDTO()
+            Id = Guid.NewGuid(),
+            Title = "Title6",
+            Phone = "1111111111",
+            WorkshopDescriptionItems = new[]
+            {
+            WithWorkshopDescriptionItem(),
+            WithWorkshopDescriptionItem(),
+            },
+            Price = 6000,
+            WithDisabilityOptions = true,
+            //Status = WorkshopStatus.Open,
+            ProviderTitle = "ProviderTitle",
+            DisabilityOptionsDesc = "Desc6",
+            Website = "website6",
+            Instagram = "insta6",
+            Facebook = "facebook6",
+            Email = "email6@gmail.com",
+            MaxAge = 10,
+            MinAge = 4,
+            //CoverImageId = "image6",
+            ProviderId = new Guid("5e519d63-0cdd-48a8-81da-6365aa5ad8c3"),
+            InstitutionHierarchyId = new Guid("af475193-6a1e-4a75-9ba3-439c4300f771"),
+            AddressId = 55,
+            Address = new AddressDto
+            {
+                Id = 55,
+                CATOTTGId = 4970,
+                Street = "Street55",
+                BuildingNumber = "BuildingNumber55",
+                Latitude = 0,
+                Longitude = 0,
+            },
+            Teachers = new List<TeacherDTO>
+        {
+            new TeacherDTO
+            {
+                Id = Guid.NewGuid(),
+                FirstName = "Alex",
+                LastName = "Brown",
+                MiddleName = "SomeMiddleName",
+                Description = "Description",
+                //Image = "Image",
+                DateOfBirth = DateTime.Parse("2000-01-01"),
+                WorkshopId = new Guid("5e519d63-0cdd-48a8-81da-6365aa5ad8c3"),
+            },
```

Comment:
(Code style) I'm bit scared seeing method with >50 lines of code. 
Hadn't checked all other DTOs, but at least this one is duplicated below in `WithWorkshopUpdate()`. 
(As a suggestion) If they could/should be similar - why not extract and reuse generating method? And do the same for other DTOs?

Reply:
I have created several test data generators for the workshop Dtos.